### ISSUE TITLE
False negative: 'image-alt' rule does not fail for alt=" "

### ISSUE THREAD

**Issue by iamrafan** [dequelabs/axe-core#1174]
- Demo: ```<img id="test" src="" alt=" ">```

- AT Tests:

  | AT/Browser | alt=" " (space)
-- | -- | --
1 | Edge/Narrator | Image
2 | Edge/JAWS | Graphic
3 | Edge/NVDA | Graphic
4 | Chrome/NVDA | Graphic
5 | Chrome/JAWS | Ignored by AT
6 | IE/JAWS | Ignored by AT
7 | IE/NVDA | Graphic
8 | Safari/voiceover | Ignored by AT
9 | Firefox/NVDA | Graphic

<pre><code>axe-core version: 3.1.1
axe-Coconut version: 3.5.0</code></pre>

**Comment by WilcoFiers** [comment]
This should probably be a separate check so that we can report the issue accurately, and so that its easy to turn off.
